platform/viewer/cypress/support/commands.js

```diff
@@ -76,12 +78,18 @@ Cypress.Commands.add('isPageLoaded', (url = '/viewer/') => {
   return cy.location('pathname', { timeout: 60000 }).should('include', url);
 });
 
-Cypress.Commands.add('openStudyList', patientName => {
+Cypress.Commands.add('openStudyList', () => {
   cy.initRouteAliases();
   cy.visit('/');
   cy.wait('@getStudies');
 });
 
+Cypress.Commands.add('waitStudyList', () => {
+  cy.get('@searchResult').should($list => {
+    expect($list).to.not.have.class('no-hover');
```

Comment:
I think checking the list length may be more reliable (if this class changes).

Reply:
The problem with length is that if we don't find results or the result is loading, the length will be 1. This can be misled if our search result also returns 1.